### ISSUE TITLE
Elapsed time uploaded file

### ISSUE THREAD

**Issue by BenyErnest** [microsoft/AzureStorageExplorer#3327]
**Describe the solution you'd like**
Hi there, It ill be nice to see the elapsed time to upload a file to blob storage, Greets.

**Comment by MRayermannMSFT** [comment]
@BenyErnest do you mean before you upload it, while you're uploading it, or after you've uploaded it?

**Comment by BenyErnest** [comment]
Hi @MRayermannMSFT,
This would be what I need:
- how long it took for a finished upload to finish

But if you can add all those features it would be better.

**Comment by MRayermannMSFT** [comment]
Ok, thanks for the clarification. 

Let's also think about if there's any other stats about uploads/downloads people might want to see once the operation is done.

**Comment by JasonYeMSFT** [comment]
This is a duplicate of #1683. We'll track this feature request together there.
